resume.html

```diff
@@ -0,0 +1,97 @@
+business network, and common network vulnerability analysis.</br>
+
+<h3>MAY 2014 - PRESENT</h3>
+
+Exploitation Analyst - USAF, San Antonio, TX & Augusta, GA</br>
+- Led six personnel across 7 projects to identify 100+ network security vulnerabilities; results
+highlighted up to and including the President of the United States.</br>
+- Senior analyst responsible for research and development against Linux/Unix variants,
+Windows, Routers, Switches, VMWare, Linux Embedded Smart Devices, Antiviruses, and
+other systems involved in network management.</br>
+- Completed over 200 live operations in pursuit of network vulnerabilities. Identified
+vulnerabilities that led to groundbreaking successes in pursuit of headquarters� objectives.</br>
+- Trained and certified as a Lead Trainer for 17 Exploitation Analysts; resulted in a 40%
+increase in qualified personnel.</br>
+- Managed six projects as Line of Effort Leader.</br>
+- Became subject matter expert. Provided numerous intelligence briefings and interoffice
+training sessions to leadership and peers to improve policies and ensure compliance;
+awarded Defense Meritorious Service Medal.</br>
+
+<h3>FEB 2012 - MAY 2014</h3>
+
+Aircraft Maintenance Quality Assurance Inspector - USAF, Okinawa, Japan</br>
+Performed periodic and spot inspections and was responsible for maintaining the Quality Control,
+Environmental and Safety Programs at the field level in support of assigned aircraft, personnel, and
+support equipment.</br>
+- Performed daily upkeep and upgrades of Microsoft Access database responsible for five
+inspection programs tracking over 47K inspections. Drafted maintenance guide distributed
+to Air Force database admins and users across the globe.</br>
+- Sole QA inspector for more than 40 personnel and 2000 avionics components. Ensured
+safety and legal compliance with local, national, and international law, earning the Air Force
+Commendation Medal.</br>
+- Maintained high attention to detail. Identified and documented over 200 safety and
+compliance discrepancies and provided corrective courses of action to responsible
+supervisors to streamline processes and prevent future occurrences.</br>
+
+<h3>OCT 2005 - MONTH 2012</h3>
+
+Avionics Test Station & Components Technician - USAF, Abilene, TX & Okinawa, Japan</br>
+Performed and managed avionics test station functions and activities. Operated, inspected,
+maintained, programmed, and calibrated computer and manually operated avionics test equipment,
+support equipment, and aircraft avionics systems components.</br>
+- Skilled troubleshooter responsible for repair and return to service of over 1000 aircraft
+parts. Saved over $500M in replacement costs through a 7-year period, earning an Air
+Force Achievement Medal.</br>
+- Lead technician responsible for software updates and system upgrades for 13
+Windows-based test stations leading to over 99% up time of required machines.</br>
```

Comment:
Is it up time or up-time?

Reply:
up-time. good catch. I've had several people look over these bullets, and nobody has caught that.